ddb_cp.module

```diff
@@ -243,19 +255,21 @@ function ddb_cp_ajax_test_result($site = NULL) {
   else {
     return MENU_NOT_FOUND;
   }
+
+  // @todo: Missing return statement it will note return any thing if the if statement is hit?
```

Comment:
To elaborate: The problem is that the DocBlock does not match the return statements in the function.

The DocBlock states:

```php
  * @return mixed
  *   JSON encoded HTML to display test result or 'Page not found'.
```

1. The function does not return JSON-encoded HTML. [`drupal_json_output`](https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_json_output/7) echoes the JSON output directly.
2. The function does not return the string 'Page not found'. The value for the constant `MENU_NOT_FOUND` is 2.